pkgpanda/docker.windows/dcos-builder/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# escape=`
+
+FROM microsoft/windowsservercore:1709
+
+#SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+
+
+# Default to PowerShell if no other command specified.
+CMD ["powershell.exe", "-NoLogo", "-ExecutionPolicy", "Bypass"]
+
+COPY ./setup-git.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-git.ps1
+
+COPY ./setup-cmake.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-cmake.ps1
+
+COPY ./setup-patch.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-patch.ps1
+
+COPY ./setup-make.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-make.ps1
+
+COPY ./setup-python27.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-python27.ps1
+#ENV PYTHON_PATH 'C:\Python27;C:\Python27\scripts'
+
+COPY ./setup-erlang.ps1 /tmp/
+RUN  powershell.exe -file /tmp/setup-erlang.ps1

```

Comment:
you build it due to an issue. we are not seeing that issue so does not make sense to build from source.

Reply:
We package it via Pkgpanda because that's how we ship all DC/OS software. Whether we build it from source is a separate issue -- there are several Pkgpanda packages where we download and tar up binaries. If we need Erlang at build and runtime, we should be distributing it as a Pkgpanda package.